### PR TITLE
Add Russian translation

### PR DESCRIPTION
**Pull request by TatianaFokina**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -59,6 +59,11 @@
                 <img class="nav__image" src="./images/flag-es.png" alt="Spanish" />
               </a>
             </li>
+            <li class="nav__list-item">
+              <a class="nav__link" href="./ru/">
+                <img class="nav__image" src="./images/flag-ru.png" alt="Russian" />
+              </a>
+            </li>
```
Comment (sergeicodes) [inline index.html]: @TatianaFokina could you put it before spanich to keep alphabetical order?

File: ru/index.html [hunk 2]
```diff
@@ -0,0 +1,495 @@
...
+            </li>
+            <li class="nav__list-item">
+              <a class="nav__link" href="../ru/">
+                <img class="nav__image" src="../images/flag-ru.png" alt="Русский" />
+              </a>
+            </li>
+          </ul>
+        </nav>
+
+        <h1>Мифы о a11y</h1>
```
Comment (sergeicodes) [inline ru/index.html]: @TatianaFokina what about `мифы о доступности` instead of `Мифы о a11y`? a11y doesn't sound good here.

Comment (sergeicodes) [reply ru/index.html]: Make sure to update `og:image` as well.

**Review by sergeicodes — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (sergeicodes) [reply index.html]: @TatianaFokina please add this navigation item to `/es/index.html` and `/jp/index.html`

File: ru/index.html (hunk 2 above)
Comment (TatianaFokina) [reply ru/index.html]: Good point!

**Pull request merged**